client/js/widgets/widget.js

```diff
@@ -672,7 +672,8 @@ export class Widget extends StateManaged {
             variables[variable] = getValue(variables[variable]);
           if(jeRoutineLogging) jeLoggingRoutineOperationSummary(a.substr(4), JSON.stringify(variables[variable]));
         } else if(a.match(new RegExp('^ *(?://.*)?\x24'))) {
-          // ignore blank and comment only lines
+          // ignore (but log) blank and comment only lines
+          if(jeRoutineLogging) jeLoggingRoutineOperationSummary(a.substr(3));
```

Comment:
This is assuming that the comment starts with a space.  `"//comment"` in a routine is logged as "//comment omment".